sdk/core/Azure.Core.Experimental/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 0.1.0-preview.5 (Unreleased)
+## 0.1.0-preview.5 (2020-09-03)
 
 ### Added
```

Comment:
We added some breaking changes for BinaryData.

Reply:
Want to push an update to this branch?